tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -299,13 +331,20 @@ def test_logs_surfacing_scheduler_basic_flow(self, cluster):
         ]  # clean log formatting
         cluster_logs = "\n".join(cluster_logs)  # make logs list into one string
 
-        assert (
-            "Performing cluster checks: potentially sending to Den, surfacing logs to Den or updating autostop."
-            in cluster_logs
+        assert "Cluster checks have failed" not in cluster_logs
+        assert "Failed to send cluster logs to Den: " not in cluster_logs
+
+        cluster_status = cluster.status()
+        logs_start_line = cluster_status.get("cluster_config").get(
+            "start_log_line", None
         )
+        logs_end_line = cluster_status.get("cluster_config").get("end_log_line", None)
```

Comment:
we could do that, but it this case we'll have to wait 1 min to ensure that the cluster check loop were executed during the test. This will cause quite latency in executing the tests....

In this case i think that calling den is less "harmful". wdyt? @jlewitt1